<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit_calc</class>
 <widget class="QDialog" name="deposit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="kapital">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 2px solid #094065;
    border-radius: 7px;
    padding: 2px;
    font-family: Arial;
background-color: rgb(198, 229, 209);
color: black;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Нет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Да</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок вклада(месяцы):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Пополнения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Налоговая ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Периодичность выплат:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sum_input">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border: 2px solid #094065;
    border-radius: 7px;
    padding: 2px;
    font-family: Arial;
    background-color: rgb(198, 229, 209);
    color: black;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="time_month">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border: 2px solid #094065;
    border-radius: 7px;
    padding: 2px;
    font-family: Arial;
    background-color: rgb(198, 229, 209);
    color: black;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="procent_input">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border: 2px solid #094065;
    border-radius: 7px;
    padding: 2px;
    font-family: Arial;
    background-color: rgb(198, 229, 209);
    color: black;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="nalog_input">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border: 2px solid #094065;
    border-radius: 7px;
    padding: 2px;
    font-family: Arial;
    background-color: rgb(198, 229, 209);
    color: black;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="popolnenya">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border: 2px solid #094065;
    border-radius: 7px;
    padding: 2px;
    font-family: Arial;
    background-color: rgb(198, 229, 209);
    color: black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма налога:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма депозита:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pocent_output">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nalog_sum_output">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_output">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="result">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(175, 207, 149);
  color: black; 
border-radius: 10px;                    
border: 2px solid #094065;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Изъятия:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="izyatiya">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border: 2px solid #094065;
    border-radius: 7px;
    padding: 2px;
    font-family: Arial;
    background-color: rgb(198, 229, 209);
    color: black;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="period_viplat">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 2px solid #094065;
    border-radius: 7px;
    padding: 2px;
    font-family: Arial;
background-color: rgb(198, 229, 209);
color: black;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
